modules/apps/transfer/spec/05_events.md

```diff
@@ -22,6 +22,7 @@ order: 5
 | fungible_token_packet | denom         | {denom}         |
 | fungible_token_packet | amount        | {amount}        |
 | fungible_token_packet | success       | {ackSuccess}    |
+| fungible_token_packet | metadata      | {metadata}      |
```

Comment:
hmmm, somehow it's clever enough to know that it needs to update this in the spec folder and not in the docs folder...